exploration/maintaining-registry.md

```diff
@@ -155,73 +169,80 @@ _What prior decisions and existing conditions limit the possible design?_
 
 _Describe the proposed solution. Consider syntax, formatting, errors, registry, tooling, interchange._
 
-The MessageFormat WG will maintain three separate function and option registries 
-beginning with the LDML46 release.
-Future updates to these registries will coincide with LDML releases.
-
-Each registry consists of a set of template-derived documents.
-Each _function_ or _option_ entry in a registry consists of a separate document.
-An _option_ entry in a given registry will not be created if there is a corresponding
-_function_ entry in the same registry for which it is an _option_.
-Proposals to include functions into the default registry
-or to include functions or options into the RGI registry
-or to include functions or options into the Unicode reserved namespace registry
-need to follow a specific process.
-
-The three registries are:
-
-1. **Default Registry**
-This is a _function_ registry.
-It includes _functions_ (and only functions) that are normatively required to be
-implemented by all implementations.
-Each function describes it's operand or operands,
+The MessageFormat WG will a set of specifications
+that standardize the implementation of functions and options in the default namespace of 
+MessageFormat v2 beginning with the LDML46 release.
+Implementations and users are strongly discourages from defining values that use
+the default namespace.
+Future updates to these sets of functions and options will coincide with LDML releases.
+
+Each _function_ is described by a single specification document.
+Each such document will use a common template.
+A _function_ can be a _formatting function_,
+a _selector_,
+or both.
+
+The specification will indicate if the _formatting function_,
+the _selector function_, or, where applicable, both are `Standard` or `Optional`.
+The specification must describe operands, including literal representations.
+
+The specification includes all defined _options_ for the function.
+Each _option_ must define which values it accepts.
+An _option_  is either `Standard` or `Optional`.
+
+_Functions_ or _options_ that have an `Optional` status
+must have a maturity level assigned.
+The maturity levels are:
+- **Proposed**
+- **Accepted**
+- **Released**
+- **Deprecated**
```

Comment:
Are these terms used by CLDR or ICU already, or are we inventing something new for MF2? My own preference would be to use instead something like alpha/beta/[no tag]/deprecated, which probably are better understood by a layperson.

I also can't figure out what the difference between "Accepted" and "Released" might be.

Reply:
"Accepted" is a specification that has been reviewed and will be part of the next LDML release (but it not part of the current one).

These terms are similar to those used by the UTC, but there's no real attempt to be rigorous about them. The names are provisional and I'm open to suggestions. alpha/beta would be okay, once proposals are accepted, but you only have to look at emoji or language subtags to see that a lot of proposals get proposed that would never reach `alpha`.

Reply:
Here's what I dug up about status labels for ICU/CLDR/Unicode

ICU:
* [ICU API statuses](https://unicode-org.github.io/icu/userguide/icu/design.html#icu-api-compatibility) - `draft`, `stable`, `deprecated`, `obsolete` (and `discouraged` in Java only?)
* `technology preview` is not defined, but has been used for a very long time ([example from ICU v4.4](https://unicode-org.github.io/icu/userguide/strings/utf-8.html#abstract-text-apis)). It is like draft, and draft is more or less like beta, but draft additionally signals to the user that there is no guarantee of stability, while draft means that the design iteration seems done so there probably won't be major changes.

CLDR: [the processes](https://cldr.unicode.org/index/process) have different labels depending on what it is. Data fields have statuses like unconfirmed. But for releases of CLDR, you have `alpha`, `beta`, and the unqualified final release itself.

Unicode data: at https://www.unicode.org/Public/, you have a `draft` subfolder that acts like the `main` trunk. Each versioned release has alpha, then beta, then the final release.